src/Components/ButtonsList/ButtonsList.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import GoodsList from '../GoodsList/GoodsList';
+
+class ButtonsList extends React.Component {
+    state = { list: this.props.goodsList };
```

Comment:
лучше делать переносы сразу, если потом захочешь еще что-то добавить
```suggestion
    state = {
      list: this.props.goodsList,
    };
```